README.md

```diff
@@ -1,5 +1,5 @@
 # Baseline Test Pages
-The Baseline and ITTP Working Group will use this repository and set of pages to store and make reference to test pages for use in validating test rules contained in the [ICT Testing Basline](https://section508coordinators.github.io/ICTTestingBaseline/). 
+The Baseline and ITTP Working Group will use this repository and set of pages to store and make reference to test pages for use in validating test rules contained in the [ICT Testing Basline](https://section508coordinators.github.io/ICTTestingBaseline/).. 
```

Comment:
I don't like this change. remove the second period.

Reply:
agree with @michaelhortongsa